resources/mixins/kubernetes/jsonnetfile.lock.json

```diff
@@ -18,7 +18,7 @@
           "subdir": "grafana-builder"
         }
       },
-      "version": "1cb605505efe5a630f00993ff611a241fdcc264f",
+      "version": "d303b2031264728728dd1e1c05f74f67027139f6",
```

Comment:
Why did you update the mixin?

Reply:
I just ran `make` and it did that all by itself, perhaps a new version?